docs/src/components/ComponentDoc/ComponentExample/ComponentExampleVariables.tsx

```diff
@@ -37,7 +36,7 @@ const ComponentExampleVariables: React.FunctionComponent<
   const { onChange, overriddenVariables, usedVariables } = props
 
   const { theme } = React.useContext<ProviderContextPrepared>(ThemeContext)
-  const [hideUnused, setHideUnused] = React.useState(true)
+  const [hideUnused] = React.useState(true)
```

Comment:
Is this just temporary and we want to invest into finding a way hot to do it or should we remove this completely? 🤔

Reply:
Let's hide this temporary and see what we can do later